### DISCUSSION TITLE
[blocks]: Discord Clone App Page

### DISCUSSION THREAD

**Discussion by TahiatXCoder** [shadcn-ui/ui#3921] [Blocks Request]
### Description

The home page you see when you go to the link:
https://discord.com/channels/@me
or
https://discord.com/app

with random servers and random people chats with random conversations and some scrollable to the top which has a long conversation.

### Example

https://discord.com/channels/@me

**Comment by ghost** [comment]
https://discord.gg/quj7DZFz

hi guys, here is a small unofficial discord channel, let's connect and share awesome things..
webpage is coming soon.. have so many things to share 🚀
